iocmanager/main_window.py

```diff
@@ -147,6 +149,10 @@ def __init__(self, hutch: str, verbose: int = 0):
         )
         self.netconfig_prep_thread.start()
 
+        # Exception Handling: show a dialog if anything in the qt main thread errors out
+        install_pydm_excepthook(use_default_handler=False)
+        self.exception_notifier = IOCExceptionNotifier(self)
```

Comment:
Are we holding onto this for a reason?  I don't see any other references to it elsewhere in this diff

Reply:
I'm just paranoid I think, I've gotten burned in the past when slots get garbage collected

Reply:
I've been there.  That's all good by me